config/rootcompute/kube-1.24/clusterrole-kubernetes-maximal-permission-policy.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  annotations:
+    bootstrap.kcp.dev/create-only: "true"
```

Comment:
Do we really want to use `createOnly` here ?

Don't we want the policy to be updated if necessary at every restart, in order to update the rules for every resource ? Never updating it seems strange to me, and will make future additions of GVRs in the `root:compute:kubernetes` APIExport quite hard.
By the way the same question stands for the `apiexport-kubernetes.k8s.io.yaml` file.
@sttts thoughts ?

Reply:
updated